kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -99,6 +99,21 @@ spec:
   clusters:
   - clusterName: cluster-1
     serverStartState: "RUNNING"
+    serverPod:
+      # Instructs Kubernetes scheduler to prefer nodes for new cluster members where there are not
+      # already members of the same cluster.
+      affinity:
```

Comment:
Is podAntiAffinity necessary for the sample? It complicates the MII sample's domain resource YAML.

If it _is_ necessary, can there instead be a single spec.configuration.podAntiAffinityEnabled or somesuch option to add the podAntiAffinity expression automagically?  With default true?  (If it is necessary it seems like it'd be a common enough issue we'd even want it be easy to configure and on by default.)

Reply:
Necessary is, of course, debatable. The reasons I went with adding the content are that a lot of customers have already hit this issue, customers blindly use samples to create their own YAML, and there aren't any functional downsides to including the content.

I don't really like the automatic insertion idea because that means this content is either hidden in a script or the template is even more complicated.  If you look at the other places where we conditionally include content in a script, the content is there in the script always but is, by default, commented out until we use `sed` to remove the commenting.